packages/manager/apps/web/client/app/hosting/database/private-sql-activation/private-sql-activation.html

```diff
@@ -40,34 +37,9 @@
                 data-items="$ctrl.versions"
                 data-match="label"
                 data-searchable
+                required
```

Comment:
```suggestion
                data-required
```

Reply:
Hello, thanks for the review, it's updated